src/frequenz/sdk/timeseries/_resampling.py

```diff
@@ -61,7 +61,7 @@
 
 ``` python
 async some_sink(Sample) -> None:
-    ...
+    pass
```

Comment:
Did this fail somehow? Even when IMHO `pass` is more semantically correct here, AFAIK `...` should work too, is just an expression (just asking out of curiosity).

Reply:
I think I was just overly cautious with my fixes, it probably was fine before